src/data/markdown/docs/02 javascript api/07 k6-experimental/04 tracing/02 Client.md

```diff
@@ -0,0 +1,94 @@
+---
+title: 'Client'
+excerpt: 'Client is a HTTP client attaching tracing information to its requests.'
+---
+
+`Client` is an HTTP client class that attaches tracing information to its requests. It lets users include a tracing context in their HTTP requests so that tracing backends (such as Grafana Tempo) can incorporate their results.
```

Comment:
```suggestion
`Client` is an HTTP client class that attaches tracing information to its requests. It lets users include a tracing context in their HTTP requests so that tracing backends (such as [Grafana Tempo](https://grafana.com/oss/tempo/)) can incorporate their results.
```